### PR TITLE
[REL-891081] - Combine integrationpoints-sdk into RIP monolith

### PR DESCRIPTION
**Pull request by sameerrelone**
- Managed Centrally Package
- Replaced package references with project references
- Dev Verification Document 
[REL-891081- Dev Verification.docx](https://github.com/relativityone/server-integrationpoints-integrationpoints/files/14520578/REL-891081-.Dev.Verification.docx)

### CODE REVIEW

File: Source/Directory.Packages.props [hunk 1]
```diff
@@ -77,7 +77,7 @@
 	<RelativityDataGrid>16.9.1</RelativityDataGrid>
     <RelativityDataTransferLegacySDK>23000.19.1</RelativityDataTransferLegacySDK>
     <RelativityDataTransferMessageService>10.2.0</RelativityDataTransferMessageService>
-    <RelativityIntegrationPointsSDK>1.3.0</RelativityIntegrationPointsSDK>
+    <RelativityIntegrationPointsServicesInterfacesPrivate>1.4.0</RelativityIntegrationPointsServicesInterfacesPrivate>
```
Comment (shivarajkamadod) [inline Source/Directory.Packages.props:80]: Is this reference required for your work? I can see it is newly added to your PR.

File: Source/Directory.Packages.props [hunk 2]
```diff
@@ -93,6 +93,8 @@
     <RelativityServicesSDK>13.5.1</RelativityServicesSDK>
     <RelativityServicesSDKPrivate>14.0.2</RelativityServicesSDKPrivate>
     <RelativityServicesInterfacesHelpers>13.5.1</RelativityServicesInterfacesHelpers>
+    <RelativityServicesLayoutInterfaces>11.4.2</RelativityServicesLayoutInterfaces>
+    <RelativitySync>0.12.1001</RelativitySync>
```
Comment (shivarajkamadod) [inline Source/Directory.Packages.props:97]: Are both the references required for your work? I can see it is newly added to your PR.

File: Source/Directory.Packages.props [hunk 3]
```diff
@@ -238,6 +242,8 @@
     <PackageVersion Include="Relativity.Services.Interfaces" Version="$(RelativityServicesSDK)" />
     <PackageVersion Include="Relativity.Services.Interfaces.Private" Version="$(RelativityServicesSDKPrivate)" />
     <PackageVersion Include="Relativity.Services.Interfaces.Helpers" Version="$(RelativityServicesInterfacesHelpers)" />
+    <PackageVersion Include="Relativity.Services.Layout.Interfaces" Version="$(RelativityServicesLayoutInterfaces)" />
+    <PackageVersion Include="Relativity.Sync" Version="$(RelativitySync)" />
```
Comment (shivarajkamadod) [inline Source/Directory.Packages.props:246]: Same here, Please avoid adding un wanted dependencies.

File: Source/Relativity.IntegrationPoints.MyFirstProvider.Provider/Relativity.IntegrationPoints.MyFirstProvider.Provider.csproj [hunk 4]
```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Relativity.IntegrationPoints.MyFirstProvider.Provider</RootNamespace>
     <AssemblyName>Relativity.IntegrationPoints.MyFirstProvider.Provider</AssemblyName>
-    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```
Comment (shivarajkamadod) [inline Source/Relativity.IntegrationPoints.MyFirstProvider.Provider/Relativity.IntegrationPoints.MyFirstProvider.Provider.csproj:12]: Please add some information about, Why the target framework version got changed. Is this not impacting anywhere?

**Review by shivarajkamadod — COMMENTED** [review]
Please uncommented functional run tests task from paske file and make sure all existing functional tests are running after **Relativity.IntegrationPoints.Contracts** and **Relativity.IntegrationPoints.SDK** removal.

File: Source/Relativity.IntegrationPoints.MyFirstProvider.Provider/Relativity.IntegrationPoints.MyFirstProvider.Provider.csproj (hunk 4 above)
Comment (sameerrelone) [reply Source/Relativity.IntegrationPoints.MyFirstProvider.Provider/Relativity.IntegrationPoints.MyFirstProvider.Provider.csproj:12]: Yes it was giving me error while building in local after merging changes into integrationpoints-sdk, it was saying .net framework version is not compatible with current framework, so current framework was 4.7, that's why i updated it

File: Source/Directory.Packages.props (hunk 3 above)
Comment (sameerrelone) [reply Source/Directory.Packages.props]: done

**Comment by sameerrelone** [comment]
> Please uncommented functional run tests task from paske file and make sure all existing functional tests are running after **Relativity.IntegrationPoints.Contracts** and **Relativity.IntegrationPoints.SDK** removal.

Done

**Review by shivarajkamadod — DISMISSED** [review]

**Review by ravira73 — APPROVED** [review]

**Review by dhanushsathukumati — APPROVED** [review]

**Pull request merged**